app/services/subscriptions/dates/weekly_service.rb

```diff
@@ -5,8 +5,6 @@ module Dates
     class WeeklyService < Subscriptions::DatesService
       WEEK_DURATION = 7
 
-      private
-
       def compute_base_date
```

Comment:
Is there a reason for removing this private keyword?

The idea was to discourage the usage of these internal method from the outside

Reply:
Well - I'm calling `compute_charges_duration` and `compute_charges_from_date` in `CreateTrueUpService` to calculate the day price... That's why 😬

Reply:
Finally updated 👍